conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+targetCompID: target
+
+# It is required param. Directory to store sequence number and message files inside the workspace directory.
+fileStorePath: store/fix/sessions
+
+# It is required param. Time of day that this FIX session becomes activated.
+# time in the format of HH:MM:SS, time is represented in UTC.
+# If the session creation time is not between StartTime and EndTime the session state will be reset (incoming / outgoing sequences and message cache will be drop).
+# The bounds move daily. That means next day after the start at 'EndTime' the bounds will be moved
+# Format of value is 99:99:99
+startTime: 00:00:00
+
+# It is required param. Time of day that this FIX session becomes deactivated.
+# time in the format of HH:MM:SS, time is represented in UTC.
+# If the session creation time is not between StartTime and EndTime the session state will be reset (incoming / outgoing sequences and message cache will be drop).
+# The bounds move daily. That means next day after the start at 'EndTime' the bounds will be moved
+# Format of value is 99:99:99
+endTime: 00:00:00
+
+# It is required param.
+# Specifies the default application version ID for the session.
+# This can either be the ApplVerID enum (see the ApplVerID field) or the BeginString for the default version.
+# Possible values: "2", "3", "4", "5", "6", "7", "8", "9"
+defaultApplVerID: 9
+
+# Determines if sequence numbers should be reset to 1 after a normal logout termination
+resetOnLogout: false
+
+# Determines if sequence numbers should be reset to 1 after an abnormal termination
+resetOnDisconnect: false
+
+# Save Heartbeats to logs or not.
+logHeartbeats: false
+
+# Check required tags or not.
+checkRequiredTags: true
+
+# If set to checked, messages must be received from the counterparty within a defined number of seconds (see MaxLatency).
+# It is useful to turn this off if a system uses localtime for it's timestamps instead of GMT.
+checkLatency: false
+
+# If CheckLatency is set to checked, this defines the number of seconds latency allowed for a message to be processed. Default is 120.
+maxLatency: 120
+
+# If set to checked, allow unknown fields in messages. This is intended for unknown fields with tags < 5000 (not user defined fields).
+allowUnknownMsgFields: false
+
+# Determines if milliseconds should be added to timestamp fields. Only available for AML 3
+millisecondsInTimeStampFields: true
+
+# Determines if microseconds should be added to timestamp fields. Only available for AML 3. This value is more priority than 'Milliseconds In Time Stamp Fields'
+microsecondsInTimeStampFields: false
+
+# Receive limit in bytes to emulate Slow Consumer
+receiveLimit: 0
+
+# Client send reject if message is invalid
+rejectInvalidMessage: true          
+
+# If set to unchecked, user defined fields will not be rejected if they are not defined in the data dictionary, or are present in messages they do not belong to.
```

Comment:
If **this is set** to **false(unchecked)**, **the** user defined fields will not be rejected if they are not defined in the data dictionary, or **in case they** are present in messages **that** they do not belong to.